inc/options/class-wpseo-option-wpseo.php

```diff
@@ -55,6 +55,7 @@ class WPSEO_Option_Wpseo extends WPSEO_Option {
 			],
 			'access_tokens' => [],
 		],
+		'started_configuration_wizard'    => false,
```

Comment:
Please evaluate with product whether this option should be added to `WPSEO_Tracking_Settings_Data`

Reply:
I guess this option should also be validated in some way, see `validate_option()` within this same class.

Reply:
Spoke with @thijsdevalk and this should indeed be moved to `WPSEO_Tracking_Settings_Data`

Reply:
I don't think it should be _moved_ 🙂 See for example similar options e.g. `enable_text_link_counter`
The default value is set in this class. Then, it's _added_ to the `WPSEO_Tracking_Settings_Data` class, meaning it's an option to collect for tracking. Not sure it should go in the `anonymous_settings` group (I don't think so) or in the `include_list` group.